apps/admin-x-activitypub/src/views/Profile/components/Posts.tsx

```diff
@@ -82,9 +82,13 @@ const Posts: React.FC<PostsProps> = ({
                             type={activity.type}
                             onClick={() => {
                                 if (activity.object.type === 'Note') {
-                                    navigate(`/notes/${encodeURIComponent(activity.object.id)}`);
+                                    navigate(`/notes/${encodeURIComponent(activity.object.id)}`, {
+                                        state: {profilePost: true}
```

Comment:
Introduced this because we navigate to `/notes/{post_apId}` from multiple components like notes feed, reader feed etc